hack/set-deployer-pipeline.sh

```diff
@@ -1,22 +1,48 @@
 #!/bin/bash
 
-set -eu
+set -eu -o pipefail
 
-# example of pushing a deployer pipeline for a cluster named CLUSTER_NAME
-# using the config at examples/cluster/sandbox.yaml
-# but overriding some of the vars
-# and setting the git resources to trigger from the current active git branch
+: "${CLUSTER_CONFIG:?}"
+: "${USER_CONFIGS:?}"
 
-PLATFORM_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+CLUSTER_NAME=$(yq -r '.["cluster-name"]' < ${CLUSTER_CONFIG})
+PIPELINE_NAME=$(yq -r '.["concourse-pipeline-name"]' < ${CLUSTER_CONFIG})
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "${CURRENT_BRANCH}" != "master" ]]; then
+	echo "WARNING: current branch is not master!!!"
+	read -p "Are you sure you want to set pipeline with platform-version pinned to ${CURRENT_BRANCH}? " -n 1 -r
+	echo
+	if [[ ! $REPLY =~ ^[Yy]$ ]]
+	then
+		[[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
+	fi
+fi
+
+approvers="/tmp/deployer-${CLUSTER_NAME}-approvers.yaml"
+echo -n "github-approvers: " > "${approvers}"
+cat ${USER_CONFIGS}/*.yaml \
```

Comment:
You may want to follow the pattern Andy introduced here. Instead of `cat`-ing the users files together, pass the collection straight to `yq`. It seems more resilient to "malformations" in the users yaml files.

`yq . ${USER_CONFIGS}/*.yaml`